sonar-python-plugin/src/main/java/org/sonar/plugins/python/IpynbNotebookParser.java

```diff
@@ -91,7 +93,9 @@ private boolean parseSourceArray(JsonParser jParser, JsonToken jsonToken) throws
     }
     while (jParser.nextToken() != JsonToken.END_ARRAY) {
       String sourceLine = jParser.getValueAsString();
-      addLineToSource(sourceLine,  jParser.currentTokenLocation());
+      var tokenLocation = jParser.currentTokenLocation();
+      colOffSet.put(aggregatedSourceLine, countEscapeCharacters(sourceLine, new LinkedHashMap<>(), tokenLocation.getColumnNr()));
+      addLineToSource(sourceLine, tokenLocation, colOffSet.get(aggregatedSourceLine));
```

Comment:
I don't think we need the colOffSet anymore. We could just do 
```suggestion
   var countEscapedChar =  countEscapeCharacters(sourceLine, new LinkedHashMap<>(), tokenLocation.getColumnNr()));
   addLineToSource(sourceLine, tokenLocation, c);
```